spec.html

```diff
@@ -72,6 +72,10 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
         1. ReturnIfAbrupt(_next_).
         1. If _next_ is *false*, then
           1. Set _iteratorRecord_.[[Done]] to *true*.
+          1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
+          1. If _remainingElementsCount_.[[Value]] is 0, then
+            1. Let _valuesArray_ be CreateArrayFromList(_values_).
```

Comment:
Should be `AggregateError` constructor call instead, though I'd prefer to fix this in subsequent PR

Reply:
I think it suffices to do

```
1. Let _error_ be a newly created `AggregateError` object.
1. Perform ! CreateDataProperty ( _error_, `"errors"`, _errorsArray_ ).
1. Return ThrowCompletion ( _error_ ).
```

([If we decide `.errors` should be non-enumerable](https://github.com/tc39/proposal-promise-any/issues/14#issuecomment-511927109) the second step will need to be replaced by the two steps
```
1. Let _errorsDesc_ be the PropertyDescriptor { [[Value]]: _errorsArray_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
1. Perform ! DefinePropertyOrThrow ( _O_, `"errors"`, _errorsDesc_ ).
```
)

Reply:
Should we leave `.errors` to be enumerable until discussion on TC39 meeting? @mathiasbynens

Reply:
SGTM, I'll add this to the slides